### PR TITLE
Update mcroni recipe

### PR DESCRIPTION
**Pull request by liampshaw**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by liampshaw** [comment]
@BiocondaBot please update

File: recipes/mcroni/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,46 @@
-{% set version = "1.0.1" %}
-{% set sha256 = "584afa62047ab9598b8fd89d0d39fbd265baf402103005d5efbbb71b440812a5" %} 
+{% set version = "1.0.0" %}
+{% set sha256 = "3e7c008f719a8447446d2706c8738beeb4975b1f" %}
```
Comment (maarten-k) [inline recipes/mcroni/meta.yaml:2]: Looks like this is not a complete checksum. Please ensure it is a complete checksum

File: recipes/mcroni/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
+
 
 package:
   name: mcroni
   version: {{ version }}
 
 source:
-  url: https://github.com/liampshaw/mcroni/archive/v{{ version }}.tar.gz
+  url: https://github.com/liampshaw/mcroni/archive/{{ version }}.tar.gz
```
Comment (maarten-k) [inline recipes/mcroni/meta.yaml:10]: missing a "v" in front of `{{ version }}` This results in a failed build.

File: recipes/mcroni/meta.yaml [hunk 3]
```diff
@@ -1,39 +1,46 @@
...
+
   sha256: {{ sha256 }}
 
 build:
   number: 0
   noarch: python
   entry_points:
     - mcroni = mcroni.mcroni:main
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
```
Comment (maarten-k) [inline recipes/mcroni/meta.yaml:19]: I think you introduce some whitespace by accident. I check before committing always with "git diff"  to see if I introduced these changes.

Comment (liampshaw) [reply recipes/mcroni/meta.yaml:17]: thanks for this and other comments! coming back to this after a while away to see if I can actually successfully merge or not

**Comment by liampshaw** [comment]
@BiocondaBot please add label

File: recipes/mcroni/meta.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
-{% set version = "1.0.1" %}
-{% set sha256 = "584afa62047ab9598b8fd89d0d39fbd265baf402103005d5efbbb71b440812a5" %} 
+{% set version = "1.0.0" %}
```
Comment (dpryan79) [inline recipes/mcroni/meta.yaml:1]: You're reverting this to an earlier version it seems. Is that intentional?

Comment (liampshaw) [reply recipes/mcroni/meta.yaml:1]: Good spot. I will do another release incrementing version and make another PR

**Comment by liampshaw** [comment]
@dpryan79 think I've fixed the version issue now, and seems to have built OK? but there may well be other issues!

**Comment by rpetit3** [comment]
Apologies, I merged the auto bump prior to seeing this one.

**Pull request closed without merging**